### DISCUSSION TITLE
how do I reset the administrator account?

### DISCUSSION THREAD

**Discussion by dhrdzy** [langgenius/dify#6680] [Help] [answered]
### Self Checks

- [X] I have searched for existing issues [search for existing issues](https://github.com/langgenius/dify/issues), including closed ones.
- [X] I confirm that I am using English to submit this report (我已阅读并同意 [Language Policy](https://github.com/langgenius/dify/issues/1542)).
- [X] 请务必使用英文提交 Issue，否则会被关闭。谢谢！:）
- [X] Please do not modify this template :) and fill in all the required fields.

### 1. Is this request related to a challenge you're experiencing? Tell me about your story.

After the local deployment is complete, how do I reset the administrator account? I can’t reset it anymore…

**Comment by crazywoola** [comment, marked as answer]
If you want to reset the email account run `flask reset-email`. 
If you want to reset the password run `flask reset-password`. 
See links here https://docs.dify.ai/learn-more/faq/install-faq#id-1.-how-to-reset-the-password-if-it-is-incorrect-after-local-deployment-initialization

If you using docker, run `docker exec -it docker-api-1 flask reset-password`

**Reply by dhrdzy** [reply]
Thank you very much!
